functions/src/main/java/com/example/functions/GcsEvent.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START functions_helloworld_gcs_event]
+package com.example.functions;
+
+public class GcsEvent {
```

Comment:
This needs to use getters (and setters).

What is the point of this class?

Reply:
This is a `struct` that incoming GCS events are marshalled into.

Reply:
Java doesn't have have the concept of `structs`. It's idiomatic to use a constructor, private attributes, and getters to make it read-only.